src/cred.c

```diff
@@ -841,6 +844,19 @@ fido_cred_set_prot(fido_cred_t *cred, int prot)
 	return (FIDO_OK);
 }
 
+int
+fido_cred_set_blob(fido_cred_t *cred, const unsigned char *ptr, size_t len)
+{
+	if (ptr == NULL || len == 0)
+		return (FIDO_ERR_INVALID_ARGUMENT);
+	else if (fido_blob_set(&cred->blob, ptr, len) < 0)
```

Comment:
you can drop the `else`

Reply:
fixed in f3271ce7be5c58e8263da49a710f8ac483cdc819